src/features/collection/components/cards/ResumeAnswerModalCard.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import formatUpdatedAt from '@/shared/utils/formatUpdateAt';
+import { Modal, ModalBody, ModalContent, ModalFooter, ModalHeader, ModalOverlay } from '@chakra-ui/react';
+import { ANSWER_MAX_LENGTH } from '../../constants';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  updatedAt: string;
+  title: string;
+  answer: string;
+};
+
+const ResumeAnswerModalCard = (props: Props) => {
+  const { isOpen, onClose, updatedAt, title, answer } = props;
```

Comment:
```suggestion
const ResumeAnswerModalCard = ({ isOpen, onClose, updatedAt, title, answer }: Props) => {
```

코드 한 줄로 줄여보는 것도 좋을 것 같아유 👀